packages/composite-checkout/src/components/checkout-steps.tsx

```diff
@@ -547,8 +550,12 @@ export const SubmitButtonWrapper = styled.div`
 			position: relative;
 		}
 
-		.rtl & {
-			padding: 24px 40px 24px 0px;
+		.checkout__step-wrapper & {
```

Comment:
This padding is meant to make the submit button align with the checkout step content and not with the checkmark/number on the side of checkout steps. This change makes it so that the padding only gets applied if we are inside a step wrapper, allowing the component to be used without the padding on its own.